pkg/local/vmis.go

```diff
@@ -99,14 +99,21 @@ func createInstance(binding *v1beta1v8o.VerrazzanoBinding, verrazzanoURI string,
 
 	storageOption := createStorageOption(enableMonitoringStorage)
 
+	var bindingName string
+	if util.IsDevProfile() {
```

Comment:
I am confused by this.  Is this PR just for the dev mode VMI where the ES pods are all master,data,and ingress?  I thought this was unrelated to that change where you want to get rid of the application specific VMI.

Reply:
Want to get rid of application specific VMI ( which is primarily in the verrazzano-monitoring-operator change ) and push app specific Prometheus metrics to the verrazzano-system Prometheus instance.  That code is in this change

The gating here is temporary and will be replaced with a verrazzano helm chart option.